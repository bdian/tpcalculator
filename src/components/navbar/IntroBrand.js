import React from 'react';
import styled from 'styled-components';
import image from "../../assets/logo.png";

const Brand = () => {
  return (
    <Image src={image} alt="TP-Calculator Logo" />
  );
};

export default Brand;

const Image = styled.img`
  display: block;
  margin: auto;
  position: absolute;
  top: 20%;
  left: 50%;
  transform: translate(-50%) }
  width: 40%;
  max-width: 400px;
  height: auto;
`;
