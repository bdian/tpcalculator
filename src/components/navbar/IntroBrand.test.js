import React from 'react';
import IntroBrand from './IntroBrand';
import renderer from 'react-test-renderer';

test('Intro Brand snapshot', () => {
  const component = renderer.create(
    <IntroBrand />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});