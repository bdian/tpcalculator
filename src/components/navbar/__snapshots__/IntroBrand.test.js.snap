// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro Brand snapshot 1`] = `
<img
  alt="TP-Calculator Logo"
  className="sc-AxjAm JKNjv"
  src={
    Object {
      "process": [Function],
    }
  }
/>
`;
